scripts/changelog_check.rb

```diff
@@ -180,7 +184,7 @@ def format_changelog(changelog_entries)
 end
 
 def main(args)
-  options = { base_branch: 'main' }
+  options = { base_branch: 'main', source_branch: current_branch }
```

Comment:
on second thought.... could we just do `HEAD` here?
```suggestion
  options = { base_branch: 'main', source_branch: 'HEAD' }
```

Reply:
Ooh, good though, let me see if that works

Reply:
Seems like it works. Much simpler, thanks 👍  Updated in 58706c653